### ISSUE TITLE
Printing the values of variables should be optional

### ISSUE THREAD

**Issue by vsajip** [motemen/gore#177]
If I do at the `gore>` prompt
```
some_var, err := some_func()
```
the values of `some_var` and `err` are always printed. Can this be made optional? If I want to type more than one command when exploring some library, for example, I don't necessarily want to clutter the display with all elements of returned structures. Of course, typing the name of a variable should display the value, as it does currently - it just shouldn't perhaps print automatically on assignments. This is related to #139, I guess.

**Comment by itchyny** [comment]
This would be a new command line flag.

**Comment by vsajip** [comment]
Yes - or more flexibly, perhaps, a new :-command such as `:verbose` that the user could invoke multiple times in a gore session, working as a toggle, or else taking an argument such as `:verbose on` /  `:verbose off`.

**Comment by itchyny** [comment]
Now implemented `:verbose` command along with `GORE_VERBOSE` environment variable.
